.buildkite/supported_plugins_test_pipeline.yml

```diff
@@ -0,0 +1,29 @@
+agents:
+  image: "docker.elastic.co/ci-agent-images/platform-ingest/buildkite-agent-logstash-ci:0.2"
+  cpu: "8"
+  memory: "16Gi"
+  ephemeralStorage: "200Gi"
+
+steps:
+  - label: "Test Tier1 inputs"
+    command: ./.buildkite/scripts/setup_java.sh && ./ci/test_supported_plugins.sh -t tier1 -k input
+  - label: "Test Tier1 filters"
+    command: ./.buildkite/scripts/setup_java.sh && ./ci/test_supported_plugins.sh -t tier1 -k filter
+  - label: "Test Tier1 codecs"
+    command: ./.buildkite/scripts/setup_java.sh && ./ci/test_supported_plugins.sh -t tier1 -k codec
+  - label: "Test Tier1 outputs"
+    command: ./.buildkite/scripts/setup_java.sh && ./ci/test_supported_plugins.sh -t tier1 -k output
+  - label: "Test Tier1 integrations"
+    command: ./.buildkite/scripts/setup_java.sh && ./ci/test_supported_plugins.sh -t tier1 -k integration
+
+  # Wait all tier1 plugins testing ends before going on tier2
+  - wait:
+
+  - label: "Test Tier2 inputs"
+    command: ./.buildkite/scripts/setup_java.sh && ./ci/test_supported_plugins.sh -t tier2 -k input
```

Comment:
if you follow the above suggested approach with groups, here you'll need to create a second group (see [example](https://buildkite.com/docs/pipelines/group-step)) and use a `depends: <group-1-key>` to run only when above succeeds.

Reply:
Solved with b6dc5d2e0b001dbffa5ee7940e15d7de3aec0536